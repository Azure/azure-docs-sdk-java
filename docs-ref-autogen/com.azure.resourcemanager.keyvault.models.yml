### YamlMime:JavaPackage
uid: "com.azure.resourcemanager.keyvault.models"
fullName: "com.azure.resourcemanager.keyvault.models"
name: "com.azure.resourcemanager.keyvault.models"
summary: "Package containing the data models for KeyVaultManagementClient."
classes:
- "com.azure.resourcemanager.keyvault.models.AccessPolicyEntry"
- "com.azure.resourcemanager.keyvault.models.ActionsRequired"
- "com.azure.resourcemanager.keyvault.models.CertificatePermissions"
- "com.azure.resourcemanager.keyvault.models.DeletedManagedHsmListResult"
- "com.azure.resourcemanager.keyvault.models.DeletedManagedHsmProperties"
- "com.azure.resourcemanager.keyvault.models.DeletedVaultListResult"
- "com.azure.resourcemanager.keyvault.models.DeletedVaultProperties"
- "com.azure.resourcemanager.keyvault.models.DimensionProperties"
- "com.azure.resourcemanager.keyvault.models.Error"
- "com.azure.resourcemanager.keyvault.models.ErrorException"
- "com.azure.resourcemanager.keyvault.models.IpRule"
- "com.azure.resourcemanager.keyvault.models.KeyPermissions"
- "com.azure.resourcemanager.keyvault.models.LogSpecification"
- "com.azure.resourcemanager.keyvault.models.ManagedHsmListResult"
- "com.azure.resourcemanager.keyvault.models.ManagedHsmProperties"
- "com.azure.resourcemanager.keyvault.models.ManagedHsmResource"
- "com.azure.resourcemanager.keyvault.models.ManagedHsmSku"
- "com.azure.resourcemanager.keyvault.models.ManagedHsmSkuFamily"
- "com.azure.resourcemanager.keyvault.models.MetricSpecification"
- "com.azure.resourcemanager.keyvault.models.MhsmNetworkRuleSet"
- "com.azure.resourcemanager.keyvault.models.MhsmPrivateEndpoint"
- "com.azure.resourcemanager.keyvault.models.MhsmPrivateEndpointConnectionItem"
- "com.azure.resourcemanager.keyvault.models.MhsmPrivateEndpointConnectionsListResult"
- "com.azure.resourcemanager.keyvault.models.MhsmPrivateEndpointConnectionsPutHeaders"
- "com.azure.resourcemanager.keyvault.models.MhsmPrivateEndpointConnectionsPutResponse"
- "com.azure.resourcemanager.keyvault.models.MhsmPrivateLinkResource"
- "com.azure.resourcemanager.keyvault.models.MhsmPrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.keyvault.models.MhsmVirtualNetworkRule"
- "com.azure.resourcemanager.keyvault.models.MhsmipRule"
- "com.azure.resourcemanager.keyvault.models.NetworkRuleAction"
- "com.azure.resourcemanager.keyvault.models.NetworkRuleBypassOptions"
- "com.azure.resourcemanager.keyvault.models.NetworkRuleSet"
- "com.azure.resourcemanager.keyvault.models.OperationDisplay"
- "com.azure.resourcemanager.keyvault.models.OperationListResult"
- "com.azure.resourcemanager.keyvault.models.Permissions"
- "com.azure.resourcemanager.keyvault.models.PrivateEndpoint"
- "com.azure.resourcemanager.keyvault.models.PrivateEndpointConnectionItem"
- "com.azure.resourcemanager.keyvault.models.PrivateEndpointConnectionListResult"
- "com.azure.resourcemanager.keyvault.models.PrivateEndpointConnectionProvisioningState"
- "com.azure.resourcemanager.keyvault.models.PrivateEndpointConnectionsPutHeaders"
- "com.azure.resourcemanager.keyvault.models.PrivateEndpointConnectionsPutResponse"
- "com.azure.resourcemanager.keyvault.models.PrivateEndpointServiceConnectionStatus"
- "com.azure.resourcemanager.keyvault.models.PrivateLinkResource"
- "com.azure.resourcemanager.keyvault.models.PrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.keyvault.models.ProvisioningState"
- "com.azure.resourcemanager.keyvault.models.PublicNetworkAccess"
- "com.azure.resourcemanager.keyvault.models.ResourceListResult"
- "com.azure.resourcemanager.keyvault.models.SecretPermissions"
- "com.azure.resourcemanager.keyvault.models.ServiceSpecification"
- "com.azure.resourcemanager.keyvault.models.Sku"
- "com.azure.resourcemanager.keyvault.models.SkuFamily"
- "com.azure.resourcemanager.keyvault.models.StoragePermissions"
- "com.azure.resourcemanager.keyvault.models.VaultAccessPolicyProperties"
- "com.azure.resourcemanager.keyvault.models.VaultCheckNameAvailabilityParameters"
- "com.azure.resourcemanager.keyvault.models.VaultCreateOrUpdateParameters"
- "com.azure.resourcemanager.keyvault.models.VaultListResult"
- "com.azure.resourcemanager.keyvault.models.VaultPatchParameters"
- "com.azure.resourcemanager.keyvault.models.VaultPatchProperties"
- "com.azure.resourcemanager.keyvault.models.VaultProperties"
- "com.azure.resourcemanager.keyvault.models.VaultProvisioningState"
- "com.azure.resourcemanager.keyvault.models.VirtualNetworkRule"
enums:
- "com.azure.resourcemanager.keyvault.models.AccessPolicyUpdateKind"
- "com.azure.resourcemanager.keyvault.models.CreateMode"
- "com.azure.resourcemanager.keyvault.models.ManagedHsmSkuName"
- "com.azure.resourcemanager.keyvault.models.Reason"
- "com.azure.resourcemanager.keyvault.models.SkuName"
interfaces:
- "com.azure.resourcemanager.keyvault.models.AccessPolicy"
- "com.azure.resourcemanager.keyvault.models.AccessPolicy.Definition"
- "com.azure.resourcemanager.keyvault.models.AccessPolicy.DefinitionStages"
- "com.azure.resourcemanager.keyvault.models.AccessPolicy.DefinitionStages.Blank"
- "com.azure.resourcemanager.keyvault.models.AccessPolicy.DefinitionStages.WithAttach"
- "com.azure.resourcemanager.keyvault.models.AccessPolicy.DefinitionStages.WithIdentity"
- "com.azure.resourcemanager.keyvault.models.AccessPolicy.DefinitionStages.WithPermissions"
- "com.azure.resourcemanager.keyvault.models.AccessPolicy.Update"
- "com.azure.resourcemanager.keyvault.models.AccessPolicy.UpdateDefinition"
- "com.azure.resourcemanager.keyvault.models.AccessPolicy.UpdateDefinitionStages"
- "com.azure.resourcemanager.keyvault.models.AccessPolicy.UpdateDefinitionStages.Blank"
- "com.azure.resourcemanager.keyvault.models.AccessPolicy.UpdateDefinitionStages.WithAttach"
- "com.azure.resourcemanager.keyvault.models.AccessPolicy.UpdateDefinitionStages.WithIdentity"
- "com.azure.resourcemanager.keyvault.models.AccessPolicy.UpdateDefinitionStages.WithPermissions"
- "com.azure.resourcemanager.keyvault.models.AccessPolicy.UpdateStages"
- "com.azure.resourcemanager.keyvault.models.AccessPolicy.UpdateStages.WithPermissions"
- "com.azure.resourcemanager.keyvault.models.CheckNameAvailabilityResult"
- "com.azure.resourcemanager.keyvault.models.DeletedVault"
- "com.azure.resourcemanager.keyvault.models.Key"
- "com.azure.resourcemanager.keyvault.models.Key.Definition"
- "com.azure.resourcemanager.keyvault.models.Key.DefinitionStages"
- "com.azure.resourcemanager.keyvault.models.Key.DefinitionStages.Blank"
- "com.azure.resourcemanager.keyvault.models.Key.DefinitionStages.WithAttributes"
- "com.azure.resourcemanager.keyvault.models.Key.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.keyvault.models.Key.DefinitionStages.WithCreateBase"
- "com.azure.resourcemanager.keyvault.models.Key.DefinitionStages.WithHsm"
- "com.azure.resourcemanager.keyvault.models.Key.DefinitionStages.WithImport"
- "com.azure.resourcemanager.keyvault.models.Key.DefinitionStages.WithKey"
- "com.azure.resourcemanager.keyvault.models.Key.DefinitionStages.WithKeyOperations"
- "com.azure.resourcemanager.keyvault.models.Key.DefinitionStages.WithKeySize"
- "com.azure.resourcemanager.keyvault.models.Key.DefinitionStages.WithTags"
- "com.azure.resourcemanager.keyvault.models.Key.Update"
- "com.azure.resourcemanager.keyvault.models.Key.UpdateStages"
- "com.azure.resourcemanager.keyvault.models.Key.UpdateStages.WithAttributes"
- "com.azure.resourcemanager.keyvault.models.Key.UpdateStages.WithHsm"
- "com.azure.resourcemanager.keyvault.models.Key.UpdateStages.WithKey"
- "com.azure.resourcemanager.keyvault.models.Key.UpdateStages.WithKeyOperations"
- "com.azure.resourcemanager.keyvault.models.Key.UpdateStages.WithKeySize"
- "com.azure.resourcemanager.keyvault.models.Key.UpdateStages.WithTags"
- "com.azure.resourcemanager.keyvault.models.Key.UpdateWithCreate"
- "com.azure.resourcemanager.keyvault.models.Key.UpdateWithImport"
- "com.azure.resourcemanager.keyvault.models.Keys"
- "com.azure.resourcemanager.keyvault.models.Secret"
- "com.azure.resourcemanager.keyvault.models.Secret.Definition"
- "com.azure.resourcemanager.keyvault.models.Secret.DefinitionStages"
- "com.azure.resourcemanager.keyvault.models.Secret.DefinitionStages.Blank"
- "com.azure.resourcemanager.keyvault.models.Secret.DefinitionStages.WithAttributes"
- "com.azure.resourcemanager.keyvault.models.Secret.DefinitionStages.WithContentType"
- "com.azure.resourcemanager.keyvault.models.Secret.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.keyvault.models.Secret.DefinitionStages.WithTags"
- "com.azure.resourcemanager.keyvault.models.Secret.DefinitionStages.WithValue"
- "com.azure.resourcemanager.keyvault.models.Secret.Update"
- "com.azure.resourcemanager.keyvault.models.Secret.UpdateStages"
- "com.azure.resourcemanager.keyvault.models.Secret.UpdateStages.WithAttributes"
- "com.azure.resourcemanager.keyvault.models.Secret.UpdateStages.WithContentType"
- "com.azure.resourcemanager.keyvault.models.Secret.UpdateStages.WithTags"
- "com.azure.resourcemanager.keyvault.models.Secret.UpdateStages.WithValue"
- "com.azure.resourcemanager.keyvault.models.Secrets"
- "com.azure.resourcemanager.keyvault.models.Vault"
- "com.azure.resourcemanager.keyvault.models.Vault.Definition"
- "com.azure.resourcemanager.keyvault.models.Vault.DefinitionStages"
- "com.azure.resourcemanager.keyvault.models.Vault.DefinitionStages.Blank"
- "com.azure.resourcemanager.keyvault.models.Vault.DefinitionStages.WithAccessPolicy"
- "com.azure.resourcemanager.keyvault.models.Vault.DefinitionStages.WithConfigurations"
- "com.azure.resourcemanager.keyvault.models.Vault.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.keyvault.models.Vault.DefinitionStages.WithGroup"
- "com.azure.resourcemanager.keyvault.models.Vault.DefinitionStages.WithNetworkRuleSet"
- "com.azure.resourcemanager.keyvault.models.Vault.DefinitionStages.WithSku"
- "com.azure.resourcemanager.keyvault.models.Vault.Update"
- "com.azure.resourcemanager.keyvault.models.Vault.UpdateStages"
- "com.azure.resourcemanager.keyvault.models.Vault.UpdateStages.WithAccessPolicy"
- "com.azure.resourcemanager.keyvault.models.Vault.UpdateStages.WithConfigurations"
- "com.azure.resourcemanager.keyvault.models.Vault.UpdateStages.WithNetworkRuleSet"
- "com.azure.resourcemanager.keyvault.models.Vaults"
desc: "Package containing the data models for KeyVaultManagementClient. The Azure management API provides a RESTful set of\n web services that interact with Azure Key Vault."
metadata: {}
package: "com.azure.resourcemanager.keyvault.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-keyvault:2.19.0
