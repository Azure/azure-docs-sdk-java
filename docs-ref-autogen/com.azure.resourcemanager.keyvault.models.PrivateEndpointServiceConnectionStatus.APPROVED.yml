### YamlMime:JavaMember
uid: "com.azure.resourcemanager.keyvault.models.PrivateEndpointServiceConnectionStatus.APPROVED*"
fullName: "com.azure.resourcemanager.keyvault.models.PrivateEndpointServiceConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.keyvault.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.keyvault.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for PrivateEndpointServiceConnectionStatus."
  syntax: "public static final PrivateEndpointServiceConnectionStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.keyvault.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-keyvault:2.1.0
